AmericanMedicalAssociation/AMA-style-guide
# EWL-3877 SG | Refactor patterns that use "Two-up-heavy-right-offset" layout and remove layout molecule

## Ticket(s)

**Jira Ticket**

- [EWL-3877: SG | Refactor patterns that use "Two-up-heavy-right-offset" layout and remove layout molecule](https://issues.ama-assn.org/browse/EWL-3877)


## Description

Removes the `two-up-heavy-right-offset` layout pattern and associated CSS.

Refactors the following patterns to use the new grid classes vs. the old layout class:
- `article`
- `article-preview-related`
- `list-topic-related-content-image`


## To Test

Visit the following patterns and make sure they don't include `.two-up-heavy-right-offset` layout classes, and also that they don't look broken: 

- [x] `article-preview-related`
- [x] `molecules-list-topic-related-articles`
- [x] `list-topic-related-content-image`
- [x] `organisms-topic-related-content-image`
- [x] `templates-article`

Visit Molecules > Layouts and verify that "Two Up Heavy Right Offset" is no longer included in the list.

## Relevant Screenshots/GIFs

N/A

## Remaining Tasks

N/A

## Additional Notes

N/A
